service/history/handler.go

```diff
@@ -527,6 +527,42 @@ func (h *Handler) StartWorkflowExecution(ctx context.Context,
 	return response, nil
 }
 
+// DescribeHistoryHost returns information about the internal states of a history host
+func (h *Handler) DescribeHistoryHost(ctx context.Context,
+	request *gen.DescribeHistoryHostRequest) (*gen.DescribeHistoryHostResponse, error) {
+	h.startWG.Wait()
+
+	numOfItemsInCacheByID, numOfItemsInCacheByName := h.domainCache.GetCacheSize()
+	status := ""
+	if h.controller.isStarted > 0 {
+		status += "started,"
+	} else {
+		status += "not started,"
+	}
+	if h.controller.isStopped > 0 {
+		status += "stopped,"
+	} else {
+		status += "not stopped,"
+	}
+	if h.controller.isStopping {
+		status += "stopping,"
```

Comment:
The comma at the end of string looks not necessary, or is that on purpose?

Reply:
Not on purpose. I will remove it.